<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlueGenstone</class>
 <widget class="QMainWindow" name="BlueGenstone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>466</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Blue Genstone</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>9</number>
    </property>
    <item>
     <widget class="QGroupBox" name="boxOfSequences">
      <property name="title">
       <string>Sequences</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sequence:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sequenceComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="sequenceAddButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add a new sequence.</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sequenceDeleteButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Delete the current sequence.</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="bitmapList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bitmapMoveUpButton">
            <property name="toolTip">
             <string>Move the select bitmap towards the beginning of the list.</string>
            </property>
            <property name="text">
             <string>Move Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bitmapMoveDownButton">
            <property name="toolTip">
             <string>Move the selected bitmap towards the end of the list.</string>
            </property>
            <property name="text">
             <string>Move Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bitmapAddButton">
            <property name="toolTip">
             <string>Add a bitmap to the list.</string>
            </property>
            <property name="text">
             <string>Add...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bitmapDeleteButton">
            <property name="toolTip">
             <string>Delete a bitmap from the list.</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="3">
        <widget class="QPushButton" name="findOutputTIFFButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Use a file picker to set the output .tif.</string>
         </property>
         <property name="text">
          <string>Find...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Output TIFF:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Dummy space color:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="outputTIFFPath">
         <property name="toolTip">
          <string>Set the path to the output .tif file.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QWidget" name="colorViewer" native="true">
         <property name="styleSheet">
          <string notr="true">* { background-color: #00FFFF} </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="dummySpaceColor">
         <property name="toolTip">
          <string>Dummy space color is used for spacing out sprites and bitmaps. Enter a RRGGBB hexadecimal code (e.g. 00FFFF) to change the dummy space color being used.</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
         <property name="placeholderText">
          <string>00FFFF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="statusText">
         <property name="toolTip">
          <string>Hi! (^)&gt;</string>
         </property>
         <property name="text">
          <string>(O)&gt; It's some debug text!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="aboutButton">
         <property name="toolTip">
          <string>Get information about this program.</string>
         </property>
         <property name="text">
          <string>About...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateTIFFButton">
         <property name="toolTip">
          <string>Generate a .tif.</string>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sequenceComboBox</tabstop>
  <tabstop>sequenceAddButton</tabstop>
  <tabstop>sequenceDeleteButton</tabstop>
  <tabstop>bitmapList</tabstop>
  <tabstop>bitmapMoveUpButton</tabstop>
  <tabstop>bitmapMoveDownButton</tabstop>
  <tabstop>bitmapAddButton</tabstop>
  <tabstop>bitmapDeleteButton</tabstop>
  <tabstop>dummySpaceColor</tabstop>
  <tabstop>outputTIFFPath</tabstop>
  <tabstop>findOutputTIFFButton</tabstop>
  <tabstop>generateTIFFButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
